src/util/redaction.ts

```diff
@@ -21,16 +24,26 @@ export interface RedactionPolicy {
 }
 
 export const DefaultConverters: Record<DefaultConvertersNames, ValueConverter> = {
-  redact: value => '[REDACTED]',
-  contextNotAllowedImage: value => 'https://impresso-project.ch/assets/images/not-allowed.png',
+  redact: _value => '[REDACTED]',
+  contextNotAllowedImage: (value, app) =>
+    app
+      ? sanitizeIiifImageUrl(
+          'https://impresso-project.ch/assets/images/not-allowed.png',
+          app.get('images').rewriteRules as ImageUrlRewriteRule[]
+        )
+      : 'https://impresso-project.ch/assets/images/not-allowed.png',
   remove: value => undefined,
   emptyArray: value => [],
 }
 
 /**
  * Redacts sensitive information from the provided object based on the specified redaction policy.
  */
-export const redactObject = <T extends Redactable>(object: T, policy: RedactionPolicy): T => {
+export const redactObject = <T extends Redactable>(
+  object: T,
+  policy: RedactionPolicy,
+  app?: ImpressoApplication
```

Comment:
Can we declare the RewriteRules type here, instead of the whole app. This will narrow the context of what the function deals with. We then get the rewrite rules from the app in `redaction.ts`

Reply:
instead of 

```
app?: ImpressoApplication
```

declare
```
rewriteRules?: ImageUrlRewriteRule[]
```